lib/v2/screens/transfer/send_enter_data/components/send_confirmation_dialog.dart

```diff
@@ -30,20 +30,34 @@ class SendConfirmationDialog extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return CustomDialog(
-      icon: SvgPicture.asset('assets/images/seeds_currency_icon.svg'),
+      icon: Stack(
+        clipBehavior: Clip.none,
+        children: [
+          SvgPicture.asset(
+            "assets/images/transfer/seeds_icon.svg",
+            height: 300,
+            width: 300,
+          ),
+          Positioned(left: 12, bottom: -6, child: SvgPicture.asset("assets/images/transfer/arrow_up.svg")),
+        ],
+      ),
       children: [
+        const SizedBox(
+          height: 6,
+        ),
         Row(
           children: [
             Text(amount, style: Theme.of(context).textTheme.headline4Black),
             Padding(
-              padding: const EdgeInsets.only(top: 12, left: 4),
-              child: Text(currency, style: Theme.of(context).textTheme.subtitle2Black),
+              padding: const EdgeInsets.only(top: 14, left: 4),
+              child: Text("SEEDS", style: Theme.of(context).textTheme.subtitle2Black),
```

Comment:
We have the word "SEEDS" as a constant in the ui_constants file. I don't know if it applies here.

Reply:
Thanks, I will try to used it